src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java

```diff
@@ -46,11 +50,15 @@
     public Covid(Person ccase) {
         this.rng = RNG.get();
         this.meanLatentPeriod = CovidParameters.get().diseaseParameters.meanLatentPeriod;
+        this.sdLatentPeriod = CovidParameters.get().diseaseParameters.sdLatentPeriod;
         this.meanAsymptomaticPeriod = CovidParameters.get().diseaseParameters.meanAsymptomaticPeriod;
+        this.sdAsymptomaticPeriod = CovidParameters.get().diseaseParameters.sdAsymptomaticPeriod;
         this.pSymptoms = CovidParameters.get().diseaseParameters.pSymptomaticCase;
+        this.pSymptomsPensioner = CovidParameters.get().diseaseParameters.pSymptomaticCasePensioner;
         this.meanSymptomDelay = CovidParameters.get().diseaseParameters.meanSymptomDelay;
         this.meanSymptomDelaySD = CovidParameters.get().diseaseParameters.meanSymptomDelaySD;
         this.meanInfectiousDuration = CovidParameters.get().diseaseParameters.meanInfectiousDuration;
+        this.sdInfectiousDuration = CovidParameters.get().diseaseParameters.sdInfectiousDuration;
         this.oPhase1Betaa = CovidParameters.get().diseaseParameters.phase1Betaa;
         this.oPhase1Betab = CovidParameters.get().diseaseParameters.phase1Betab;
         this.asymptomaticTransAdjustment = CovidParameters.get().diseaseParameters.aSymptomaticTransAdjustment;
```

Comment:
Do we need to store all these variables as class fields? The might be better as local variables when we, for example, set the periods (it's not a big issue but will mean we carry around less data and hopefully improve readability by making it clear the important bits that are set in the constructor).

We don't need to change this for this PR, but probably something to keep in mind.

Reply:
To be honest, I think we should probably read them in as parameters, if we are specifying a mean and the standard deviation is not 1 thne we should be explicit about it